examples/benchmarks/TFT/tft.py

```diff
@@ -289,3 +291,24 @@ def finetune(self, dataset: DatasetH):
             dataset for finetuning
         """
         pass
+
+    def to_pickle(self, path: Union[Path, str]):
+        """
+        Tensorflow model can't be dumped directly.
+        So the data should be save seperatedly
+
+        **TODO**: Please implement the function to load the files
+
+        Parameters
+        ----------
+        path : Union[Path, str]
+            the target path to be dumped
+        """
+        # save tensorflow model
+        path = Path(path)
```

Comment:
1. insert this `from IPython import embed; embed()`
2. run and wait
2. coding try to save model